src/Consolonia.AvaloniaEdit/Caret.cs

```diff
@@ -1,3 +1,4 @@
+using System;
 using Avalonia;
 using Avalonia.Media;
 using AvaloniaEdit;
```

Comment:
Looks like caret is not native. It covers the entire character and hides it 
<img width="327" height="210" alt="image" src="https://github.com/user-attachments/assets/78a870c8-3006-4742-a4dc-9e79a8559a33" />
I guess we should introduce caret similar to TextBox